Dockerfile

```diff
@@ -5,6 +5,7 @@ RUN apt-get install -y dpkg-dev zlib1g-dev libssl-dev libffi-dev
 RUN apt-get install -y curl libcurl4-openssl-dev
 
 ENV R_REMOTES_NO_ERRORS_FROM_WARNINGS=true
+ENV CRYPTOGRAPHY_DONT_BUILD_RUST=true
```

Comment:
naive question: is this necessary even with the recent update @jkiang13 made to synapse python client? synapser wraps that iirc.

Reply:
The Python client 2.3.1 fixes this and the next synapser (0.10) due to be released shortly will embed that version and embed an upgraded version of pip that also avoids this (more recent versions of PIP can generally install cryptography from a more recent wheel which doesn't require compilation either anyway). Installing synapser < 0.10 may still need this depending on the environmental conditions.